src/System.Buffers.Experimental/System/Buffers/Text/BufferWriter_writable.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers.Text;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Text.Utf8;
+
+namespace System.Buffers.Text
+{
+    public ref partial struct BufferWriter
+    {
+        #region Byte
+        public bool TryWriteBytes(byte[] bytes)
+            => TryWriteBytes(bytes.AsReadOnlySpan());
+
+        public void WriteBytes(byte[] bytes)
+            => WriteBytes(bytes.AsReadOnlySpan());
+
+        public bool TryWriteBytes(ReadOnlySpan<byte> bytes)
+        {
+            if (bytes.TryCopyTo(Free))
+            {
+                _written += bytes.Length;
+                return true;
+            }
+            return false;
+        }
+
+        public void WriteBytes(ReadOnlySpan<byte> bytes)
+        {
+            while (!TryWriteBytes(bytes)) Resize();
+        }
+
+        public bool TryWriteBytes(ReadOnlyMemory<byte> bytes)
+            => TryWriteBytes(bytes.Span);
+
+        public void WriteBytes(ReadOnlyMemory<byte> bytes)
+            => WriteBytes(bytes.Span);
+
+        #endregion
+
+        #region IWritable
+        public bool TryWriteBytes<T>(T value, StandardFormat format) where T : IWritable
+        {
+            int written;
```

Comment:
nit: inline declaration of out variable?

Reply:
will do